config/initializers/datadog.rb

```diff
@@ -30,5 +30,36 @@
     # Enable ASM
     c.appsec.enabled = true
     c.appsec.instrument :rails
+
+    # Add dogstatsd_mapper_profiles field
+    c.dogstatsd_mapper_profiles = [
+      {
+        name: 'sidekiq',
+        prefix: 'sidekiq.',
+        mappings: [
+          {
+            match: 'sidekiq.sidekiq.*',
+            match_type: 'regex',
+            name: 'sidekiq.$1'
```

Comment:
Why does this match not have a tag but the others do?

Reply:
Not sure. This comes from this [datadog agent/sidekiq docs](https://docs.datadoghq.com/integrations/sidekiq/):

```
# dogstatsd_mapper_cache_size: 1000  # default to 1000
dogstatsd_mapper_profiles:
  - name: sidekiq
    prefix: "sidekiq."
    mappings:
      - match: 'sidekiq\.sidekiq\.(.*)'
        match_type: "regex"
        name: "sidekiq.$1"
      - match: 'sidekiq\.jobs\.(.*)\.perform'
        name: "sidekiq.jobs.perform"
        match_type: "regex"
        tags:
          worker: "$1"
      - match: 'sidekiq\.jobs\.(.*)\.(count|success|failure)'
        name: "sidekiq.jobs.worker.$2"
        match_type: "regex"
        tags:
          worker: "$1"
```